qa/tasks/ses_qa.py

```diff
@@ -149,17 +149,12 @@ def drive_replace_initiate(self, **kwargs):
         total_osds = self.master_remote.sh(number_of_osds_in_cluster)
         osd_id = 0
         disks = self._get_drive_group_limit()
-        assert int(total_osds) == disks, "Unexpected number of osds {} (expected {})".format(total_osds, disks)
-        self.master_remote.sh("sudo ceph osd tree --format json | tee before.json")
-        osd_path = self.master_remote.sh("sudo salt '*' cephdisks.find_by_osd_id {} --out json | jq -j '.[][].path'".format(osd_id))
-        self.master_remote.sh("sudo salt-run osd.replace {}".format(osd_id))
-        self.master_remote.sh("""
-          sudo sgdisk -Z {path} &&
-          sudo mkfs.ext4 -F {path} &&
-          sudo mount {path} /mnt && sync && lsblk &&
-          sudo ceph-volume inventory""".format(path=osd_path))
-        self.master_remote.sh("sudo salt-run disks.c_v_commands 2>/dev/null")
-        # Output is like: ceph-volume lvm batch --no-auto /dev/vdb --yes --osd-ids 2
+        assert int(total_osds) == disks, "Unexpected number of osds {} (expected {})"\
+            .format(total_osds, disks)
+        self.scripts.run(
+                self.master_remote,
+                'drive_replace.sh', args=[osd_id]
+                )
```

Comment:
nit - please move `args=[osd_id]` onto a line by itself, so each argument to `run` is on its own line

(for code style uniformity)

Reply:
Thanks, should be fixed